packages/ckeditor5-paragraph/src/insertparagraphcommand.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module paragraph/insertparagraphcommand
+ */
+
+import Command from '@ckeditor/ckeditor5-core/src/command';
+
+/**
+ * The insert paragraph command. It insert a new paragraph next to the provided model element.
+ *
+ *		// Insert a new paragraph before the first element in the document root.
+ *		editor.execute( 'insertParagraph', {
+ *			element: editor.model.document.getRoot().getChild( 0 ),
+ *			position: 'before'
+ *		} );
+ *
+ * **Note**: This command moves the selection to the inserted paragraph.
+ *
+ * @extends module:core/command~Command
+ */
+export default class InsertParagraphCommand extends Command {
+	/**
+	 * Executes the command.
+	 *
+	 * @protected
```

Comment:
Why is this protected, it's public in base class.

Reply:
Copy&paste from https://github.com/ckeditor/ckeditor5/blob/i/6775/packages/ckeditor5-highlight/src/highlightcommand.js#L44-L47. Then it's wrong here too :)